app/javascript/dashboard/components/widgets/conversation/components/SLACardLabel.vue

```diff
@@ -55,19 +57,27 @@ export default {
   data() {
     return {
       timer: null,
-      slaStatus: {},
+      slaStatus: {
+        threshold: null,
+        isSlaMissed: false,
+        type: null,
+        icon: null,
+      },
     };
   },
   computed: {
-    ...mapGetters({
-      activeSLA: 'sla/getSLAById',
-    }),
     slaPolicyId() {
       return this.chat?.sla_policy_id;
     },
-    sla() {
+    appliedSLA() {
       if (!this.slaPolicyId) return null;
```

Comment:
This check is unnecessary because we only display this component if `sla_policy_id` exists.